streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -260,23 +271,13 @@ public StateStore getStore(final String name) {
         // return the current offsets for those logged stores
         final Map<TopicPartition, Long> changelogOffsets = new HashMap<>();
         for (final StateStoreMetadata storeMetadata : stores.values()) {
-            if (storeMetadata.changelogPartition != null && storeMetadata.offset != null) {
+            if (storeMetadata.changelogPartition != null) {
```

Comment:
Here we still want to return the changelog -> offset entry even if it is null (indicating the current position unknown).